renovate.json

```diff
@@ -1,25 +1,16 @@
 {
-  "dependencyDashboard": true,
-  "extends": [
-    "config:base"
-  ],
+  "extends": ["config:base", "group:allNonMajor", "schedule:weekly"],
   "labels": ["Maintenance 🔨"],
   "packageRules": [
     {
+      "groupName": "internal dependencies",
+      "groupSlug": "internal",
       "packagePatterns": [
```

Comment:
I can't find `packagePatterns` in the docs, do you happen to know what this rule does?

Reply:
It looks like it's been replaced by `matchPackagePatterns`, so would this only group the packages below?

Reply:
I've updated this, we'll have to see if it still works after landing :) 

The previous approach is working on the other sites still so it might just be undocumented now. In any event if this works then we can make this change elsewhere.